Lowercase "and" becomes a part of the step definition

I started writing steps with a lowercase "and" as it splits out the sections nicely for me:

```gherkin
Given that I'm writing an example
and that I like the lowercase 'and'
When I run the pytest-bdd runner
Then it should find the second given step
```

but I noticed that the runner looks for a **given** starting with a lowercase "and":
```
E               StepDefinitionNotFoundError: Step definition is not found: Given "and that I like the lowercase 'and'"
``` 

So that the step definition I expect to match it doesn't:

```python
@given("that I like the lowercase 'and'")
def lower_case_and():
    # ...
    return
```

I can't think of any scenarios where this behaviour would be desirable. Should the runner not take both lower case and capitalised "and"s as being step definers?

--- a/tests/feature/test_steps.py
+++ b/tests/feature/test_steps.py
@@ -1,4 +1,5 @@
 import textwrap
+from idlelib.iomenu import errors
 
 
 def test_steps(pytester):
@@ -620,3 +621,48 @@ Feature: A feature
             "*Tearing down...*",
         ]
     )
+
+
+def test_lower_case_and(pytester):
+    pytester.makefile(
+        ".feature",
+        steps=textwrap.dedent(
+            """\
+            Feature: Step keywords need to be capitalised
+
+                Scenario: Step keywords must be capitalised
+                    Given that I'm writing an example
+                    and that I like the lowercase 'and'
+                    Then it should fail to parse
+            """
+        ),
+    )
+    pytester.makepyfile(
+        textwrap.dedent(
+            """\
+        import pytest
+        from pytest_bdd import given, when, then, scenarios
+
+        scenarios("steps.feature")
+
+
+        @given("that I'm writing an example")
+        def _():
+            pass
+
+
+        @given("that I like the lowercase 'and'")
+        def _():
+            pass
+
+
+        @then("it should fail to parse")
+        def _():
+            pass
+
+        """
+        )
+    )
+    result = pytester.runpytest()
+    result.assert_outcomes(errors=1)
+    result.stdout.fnmatch_lines("*TokenError*")

--- a/tests/feature/test_steps.py
+++ b/tests/feature/test_steps.py
@@ -1,5 +1,4 @@
 import textwrap
-from idlelib.iomenu import errors
 
 
 def test_steps(pytester):

